policies/coding-style.md

```diff
@@ -591,21 +591,89 @@ should be used. This is more commonly referred to as C90. ISO/IEC 9899:1999
 (also known as C99) is not supported on some platforms that OpenSSL is
 used on and therefore should be avoided.
 
-## Chapter 15: Miscellaneous
+## Chapter 15: Expressions
 
-Do not use `!` to check if a pointer is NULL, or to see if a str...cmp
-function found a match.  For example, these are wrong:
+Avoid needless parentheses.
```

Comment:
As in I would remove the "Avoid needless parentheses" entirely ...

Reply:
Why? Maybe make it a little less imperative? `Needless parentheses should be avoided.`

Reply:
Actually the current wording is `Avoid needless parentheses as far as reasonable.` And IMO that's OK.